gito/cli.py

```
@@ -1,22 +1,34 @@
-from .utils import no_subcommand, parse_refs_pair
+from .constants import HOME_ENV_PATH, GITHUB_MD_REPORT_FILE_NAME
+from .bootstrap import bootstrap
+from .utils import no_subcommand, extract_gh_owner_repo, remove_html_comments
+from .gh_api import resolve_gh_token
 
 # Import fix command to register it
-from .commands import fix, gh_post_review_comment, gh_react_to_comment, repl, deploy  # noqa
-
+from .commands import fix, gh_react_to_comment, repl, deploy, linear_comment  # noqa
+from .commands.gh_post_review_comment import post_github_cr_comment
+from .commands.linear_comment import linear_comment
```

Ruff (0.11.9)

Redefinition of unused `linear_comment` from line 29

(F811)